src/main/java/com/fasterxml/jackson/databind/deser/impl/BeanPropertyMap.java

```diff
@@ -713,25 +713,6 @@ public String toString()
     /**********************************************************
      */
 
-    protected SettableBeanProperty _rename(SettableBeanProperty prop, NameTransformer xf)
```

Comment:
this breaks backward compatbility - add back the old method and deprecate it

Reply:
the method can call the new unwrap method, but we need to keep the existing method (its implementation can change - as long as it the change retains the existing behaviour)

Reply:
good point, readded it!

Reply:
Looking at this again, while I usually agree, in this case it is an internal method called from nowhere else (and `BeanPropertyMap` not extensible via sub-calling). But it's ok to leave it for now, being just 1-liner.